src/account/account.repository.knex.ts

```diff
@@ -24,8 +24,12 @@ export class KnexAccountRepository {
                 name: account.name,
                 bio: account.bio,
                 username: account.username,
-                avatar_url: account.avatarUrl?.href,
-                banner_image_url: account.bannerImageUrl?.href,
+                avatar_url:
+                    account.avatarUrl == null ? null : account.avatarUrl?.href,
```

Comment:
Could also be written as `account.avatarUrl?.href ?? null`

Reply:
Nice one, updated in https://github.com/TryGhost/ActivityPub/pull/562/commits/3635a0125d946489ab815b0ee7a66632d365ce67